app/controllers/projects/new_tree_controller.rb

```diff
@@ -1,16 +1,66 @@
 class Projects::NewTreeController < Projects::BaseTreeController
   before_filter :require_branch_head
   before_filter :authorize_push_code!
-
+   
   def show
   end
 
   def update
-    file_path = File.join(@path, File.basename(params[:file_name]))
-    result = Files::CreateService.new(@project, current_user, params, @ref, file_path).execute
+    flag = 0
+    #replace & upload 
+    if params[:file_upload] != nil
+    #get the name of the upload file
```

Comment:
Incorrect indentation detected (column 4 instead of 6).<br>Missing space after #.